client/public/static/js/recipes/recipes.js

```diff
@@ -1,5 +1,9 @@
 const recipesView = new RecipesView();
+const publicUserRecipesPerPage = 5; // Number of public recipes per page
 let showPublicRecipes = true; // Global toggle variable for showing/hiding public recipes
+let isFirstPage = true; // Flag to indicate if it's the first page load
```

Comment:
Do we still need to check if its the first page? I don't think this is needed since we will have user recipes on multiple pages.